processor/src/test/java/org/mapstruct/ap/test/versioninfo/SuppressTimestampViaMapper.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.versioninfo;
+
+import org.mapstruct.Mapper;
+
+/**
+ * @author Filip Hrisafov
+ */
+@Mapper(suppressGeneratorTimestamp = true)
```

Comment:
I guess, we our fixture checker somehow ignores the TS as well? That code can probably be removed, right?

Reply:
Yes our fixture checker ignores the TS and the comment, because they are different each time and also they are different on different compilers.

Which code are you referring to? The one that does the ignoring? We still need to ignore the comment though.

Reply:
agreed. but not the TS.. I did not search the code.. But logic demands there would be code  that's irrelevant. Anyway.. its not a big deal because removing this would also require updating all fixtures.

Reply:
`JavaFileAssert#ignoreDelta`

Reply:
It means updating all fixtures and all our tests to actually suppress the timestamp generation

Reply:
That's what I meant in my comment 12 min ago.. 😉